pkg/apis/policy/validation/validation.go

```diff
@@ -70,8 +71,14 @@ func ValidatePodDisruptionBudgetSpec(spec policy.PodDisruptionBudgetSpec, fldPat
 
 // ValidatePodDisruptionBudgetStatus validates a PodDisruptionBudgetStatus and returns an ErrorList
 // with any errors.
-func ValidatePodDisruptionBudgetStatus(status policy.PodDisruptionBudgetStatus, fldPath *field.Path) field.ErrorList {
+func ValidatePodDisruptionBudgetStatus(status policy.PodDisruptionBudgetStatus, fldPath *field.Path, apiVersion string) field.ErrorList {
```

Comment:
pass API version as `schema.GroupVersion` to make it clear this is a fully-qualified group/version

Reply:
Updated.